### PR TITLE
hwdb: 60-keyboard:: merge Micro Star in MSI and  merge MSI Prestige And MSI Modern FnKeys and Special keys definitions

### PR DESCRIPTION
**Pull request by sainak**
* Merge Micro star and MSI as it's the same brand MSI.
* MSI Prestige and MSI Modern have similar keymaps for all models (AMD/Intel 14/15 inch)

'A10' was the model name for prestige series with 10th gen intel processors and 'SC' was the gpu config for a specific model

but now msi has refreshed these series with 11th gen intel processors so the model name becomes 'A11X' where x is gpu config name more info [here](https://www.msi.com/Business-Productivity/Prestige-14-A11X/Specification) and [here](https://www.msi.com/Business-Productivity/Prestige-15-A11X/Specification) 
[this was the link for model comparison of 10th gen cpus](https://www.msi.com/Business-Productivity/Prestige-15-A10X/Specification) (Dead link now)
complete list of Prestige series laptops [here](https://www.msi.com/Business-Productivity/Products#?tag=Prestige-Series)

And modern series follows the same naming convention full list of laptops [here](https://www.msi.com/Business-Productivity/Products#?tag=Modern-Series)

examples

Prestige series
Prestige 14 A10x ( 14inch Intel 10th gen )
Prestige 15 A11x ( 15inch Intel 11th gen )

First gen Modern Series
Modern 15 A10x  ( 15inch Intel 10th gen)
Modern 15 A4x ( 15inch AMD 4000 series)

Second gen Modern Series
Modern 15 B10x  ( 15inch Intel first gen modern laptop)
Modern 14 B4x ( 14inch AMD second gen modern laptop)

all these laptops have the same keyboard layouts with same functionality on windows (can be reviewed from their support manuals).

### CODE REVIEW

File: hwdb.d/60-keyboard.hwdb [hunk 1]
```diff
@@ -1251,7 +1251,7 @@ evdev:input:b0003v1532p0200*
  KEYBOARD_KEY_c01cb=down                                # zoomout
 
 ###########################################################
-# Micro Star
+# MSI
```
Comment (poettering) [inline hwdb.d/60-keyboard.hwdb]: maybe say `MSI (aka "Micro Star")` here then

File: hwdb.d/60-keyboard.hwdb [hunk 2]
```diff
@@ -1290,24 +1290,19 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*PR200*:*
...
- KEYBOARD_KEY_f2=f21
-
-# MSI Modern series
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-StarInternational*:pnModern*:*
- KEYBOARD_KEY_56=backslash                              # secondary backslash key
- KEYBOARD_KEY_f1=f20                                    # Fn+F5 micmute
- KEYBOARD_KEY_76=f21                                    # Fn+F4 touchpad, becomes meta+ctrl+toggle
+# Keymaps MSI Prestige And MSI Modern FnKeys and Special keys
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*Prestige*:*
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*Modern*:*
```
Comment (poettering) [inline hwdb.d/60-keyboard.hwdb:1295]: where did the match for A10SC go?

Comment (sainak) [reply hwdb.d/60-keyboard.hwdb:1295]: updated pr comment, sorry for not explaining earlier

File: hwdb.d/60-keyboard.hwdb (hunk 1 above)
Comment (sainak) [reply hwdb.d/60-keyboard.hwdb]: Okay

**Comment by sainAk** [comment]
@poettering I need your guidance on something
my laptop (MSI modern 14 B4MW) has LEDs for mute and micmute (other laptops have it too) but doesn't work on linux.
I am able to light up the LED by writing values directly to the EC. 
How can I make it work as intended, please tell me if you from where I should start.

the LED on caps lock works fine

```shell
$ ls -l /sys/class/leds/
total 0
lrwxrwxrwx 1 root root 0 Apr 15 10:37 input4::capslock -> ../../devices/platform/i8042/serio0/input/input4/input4::capslock
lrwxrwxrwx 1 root root 0 Apr 15 10:37 input4::numlock -> ../../devices/platform/i8042/serio0/input/input4/input4::numlock
lrwxrwxrwx 1 root root 0 Apr 15 10:37 input4::scrolllock -> ../../devices/platform/i8042/serio0/input/input4/input4::scrolllock
lrwxrwxrwx 1 root root 0 Apr 15 11:04 mmc0:: -> ../../devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-4/1-4:1.0/rtsx_usb_sdmmc.1.auto/leds/mmc0::
```
And I dont have a scroll lock or num lock but it's still showing up here.

File: hwdb.d/60-keyboard.hwdb (hunk 2 above)
Comment (keszybz) [reply hwdb.d/60-keyboard.hwdb:1295]: Please put the comment in the commit. Nobody looks at the PR comments once the PR is merged.

File: hwdb.d/60-keyboard.hwdb [hunk 3]
```diff
@@ -1251,7 +1251,7 @@ evdev:input:b0003v1532p0200*
  KEYBOARD_KEY_c01cb=down                                # zoomout
 
 ###########################################################
-# MSI
+# MSI (aka "Micro Star")
```
Comment (keszybz) [inline hwdb.d/60-keyboard.hwdb:1254]: Please squash this commit with the first one.

**Review by keszybz — CHANGES_REQUESTED** [review]

**Comment by keszybz** [comment]
LGTM.

**Pull request merged**